crates/utils/src/utils/validation.rs

```diff
@@ -24,6 +24,9 @@ const BIO_MAX_LENGTH: usize = 300;
 const SITE_NAME_MAX_LENGTH: usize = 20;
 const SITE_NAME_MIN_LENGTH: usize = 1;
 const SITE_DESCRIPTION_MAX_LENGTH: usize = 150;
+const FORBIDDEN_DISPLAY_CHARS: [char; 7] = [
+  '@', '\u{180e}', '\u{200b}', '\u{2060}', '\u{2800}', '\u{3164}', '\u{ffef}',
```

Comment:
Do you have a reference for this list? The site below lists even more.

https://invisible-characters.com/

Reply:
Please include a comment with the link as well, and mention that these are specifically invisible chars.